aws/table_aws_ssm_inventory_entry.go

```diff
@@ -0,0 +1,151 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/service/ssm"
+	ssmv1 "github.com/aws/aws-sdk-go/service/ssm"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+func tableAwsSSMInventoryEntry(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ssm_inventory_entry",
+		Description: "AWS SSM Inventory Entry",
+		List: &plugin.ListConfig{
+			ParentHydrate: listAwsSSMInventories,
+			Hydrate:       listAwsSSMInventoryEntries,
+			KeyColumns: plugin.KeyColumnSlice{
```

Comment:
@ParthaI the API supports `Filters` as an input parameter. Should we add this as an optional qual?
https://docs.aws.amazon.com/systems-manager/latest/APIReference/API_ListInventoryEntries.html

Reply:
@misraved In the `Filters` parameter we can pass the Instance Id, the same we can achieve by using `instance_id` optional key qual. Here we have added the the `instance_id` column as optional qual so we have not added the `Filters` parameter.